app/helpers/action_page_helper.rb

```diff
@@ -38,6 +38,9 @@ def email_friends_url(action_page)
     "mailto:?subject=#{u subject}&body=#{u message}"
   end
 
+  def link_share_url(action_page)
```

Comment:
What's the purpose of this helper? Seems like maybe we're getting the share url in javascript in `copy-url.js`.

Reply:
You're right, that's a remnant of another thought path I went down.